plugins/kotlin/uast/uast-kotlin/test/org/jetbrains/uast/test/comparasion/FE1UastApiTest.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+package org.jetbrains.uast.test.kotlin.org.jetbrains.uast.test.comparasion
+
+import com.intellij.testFramework.TestDataPath
+import org.jetbrains.kotlin.idea.test.JUnit3RunnerWithInners
+import org.jetbrains.kotlin.test.TestMetadata
+import org.jetbrains.uast.UFile
+import org.jetbrains.uast.test.common.kotlin.UastApiTestBase
+import org.jetbrains.uast.test.kotlin.org.jetbrains.uast.test.env.kotlin.AbstractFE1UastTest
+import org.junit.runner.RunWith
+import java.io.File
+
+@RunWith(JUnit3RunnerWithInners::class)
+class FE1UastApiTest : AbstractFE1UastTest() {
+    override fun check(testName: String, file: UFile) {
+        // Bogus
+    }
+
+    @TestMetadata("uast-kotlin/testData")
+    @TestDataPath("/")
+    class Legacy : AbstractFE1UastTest(), UastApiTestBase {
+        override var testDataDir = File("../uast-kotlin/testData").normalize()
```

Comment:
Could you please specify a path relative to a `KotlinRoot.DIR` (https://github.com/JetBrains/intellij-community/blob/kt-212-master/plugins/kotlin/tests-common/test/org/jetbrains/kotlin/test/KotlinRoot.kt#L16) 

similar to https://github.com/JetBrains/intellij-community/blob/kt-212-master/plugins/kotlin/uast/uast-kotlin-fir/test/org/jetbrains/kotlin/idea/fir/uast/FirUastResolveApiTest.kt#L18

This will help tests on the TC

Reply:
Done (also for existing resolve API tests)